go.mod

```diff
@@ -78,6 +78,12 @@ require (
 	sigs.k8s.io/controller-runtime v0.11.0
 )
 
+require (
```

Comment:
Hmm, is this a 3rd require section? I think we only want 2 (direct and indirect)...

Reply:
Not sure why a third section was added here. I will consolidate it back to two sections.